common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationAwareUTF8String.java

```diff
@@ -1267,6 +1268,119 @@ public static UTF8String[] icuSplitSQL(final UTF8String string, final UTF8String
     return strings.toArray(new UTF8String[0]);
   }
 
+  /**
+   * Title casing a string according to a new behaviour.
+   * Iterates over the string and title cases the first character in each word, and lowercases
+   * every other character.
+   * Handles lowercasing greek sigma(Σ) separately, taking into account if
```

Comment:
```suggestion
   * Handles lowercasing greek sigma (Σ) separately, taking into account if
```
we should try to use proper grammar in the comments

Reply:
(to avoid duplicate comments on the PR - this goes for all instances below)